src/main/java/core/basesyntax/ColorEnum.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum ColorEnum {
+    Blue,
+    Yellow,
+    Green,
+    White,
+    Red

```

Comment:
Let's name them in uppercase, don't forget about conventions. See [here](https://www.digitalocean.com/community/tutorials/java-enum).